contents/blog/ab-testing-examples.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "5 A/B testing examples from YC’s top companies"
```

Comment:
```suggestion
title: "Practical A/B testing examples from Airbnb and YC's top companies"
```

I've obsessed over this damn headline for 40 mins and this is my answer. All good?

Reply:
To explain a little bit, I always feel people are less likely to click on a list of 5 because it's the most generic list length – odd numbers are better, i.e. 7, 12, 9 and so on. "In-depth" conveys quality and Airbnb adds name recog. I did play with dropping YC and for something like "In-depth A/B testing examples from Airbnb, Instacart, and more", but I think this works.

If you're happy, feel free to merge.